{{cookiecutter.repostory_name}}/pyproject.toml

```diff
@@ -90,19 +87,20 @@ line-length = 120
 [tool.ruff.lint]
 # TODO add D
 select = [
-    "E", "F", "I", "UP",
+    "E", "F", "I", "UP", "S",
     "TCH005",
 ]
 # TODO: remove E501 once docstrings are formatted
 ignore = [
     "D100", "D105", "D107", "D200", "D202", "D203", "D205", "D212", "D400", "D401", "D415",
     "D101", "D102","D103", "D104", # TODO remove once we have docstring for all public methods
     "E501", # TODO: remove E501 once docstrings are formatted
+    "S101", "S108", "S603", "S607",
 ]
 
 [tool.ruff.lint.per-file-ignores]
 "__init__.py" = ["F401"]
-"test/**" = ["D", "F403", "F405"]
+"**/test*/**.py" = ["D", "F403", "F405", "S106", "S311"]
```

Comment:
I believe previous pattern didn't work for many cases, e.g.:
`tests/test_something.py`